src/librbd/api/Mirror.cc

```diff
@@ -2535,6 +2535,10 @@ int prepare_group_images(IoCtx& group_ioctx,
         lderr(cct) << "cannot enable mirroring: image is in a different pool"
                    << dendl;
         return -ENOTSUP;
+      } else if (image_ctx->parent != nullptr) {
+        lderr(cct) << "cannot enable mirroring: group cannot contain clone images"
+                   << dendl;
+        return -ENOTSUP;
```

Comment:
Note to self, this need to tested and everything looks good, this commit needs to dropped as per the plan.